infrastructure/validators/build-a-community.md

```diff
@@ -0,0 +1,278 @@
+
+It's also helpful to outline what kind of OS is used on these servers and what is the updating
+policy for the software on that OS. For example, are LTS versions used? Do they use NiXOS,
+distro-packaged libraries? Any server hardening practices, etc.
+
+#### Specs
+
+Are you running the recommended Standard Hardware for a validator? Can you ensure that machines have
+enough processing power, memory, file storage, and network connectivity? It's helpful for nominators
+to know the specs of the machines a validator uses to assess how they may perform in the network. If
+a validator is running underpowered machines, they may not want to nominate them, as these can
+result in fewer blocks produced and fewer overall rewards. In certain circumstances, more powerful
+machines can result in higher rewards for both the Validator and their Nominators.
+
+#### Automation and orchestration approaches (Terraform, Ansible, Chef, Puppet, Kubernetes, etc.)
+
+What kind of approach is taken for spinning up and provisioning nodes? How might you automate
+spinning up large clusters of nodes and upgrading them? Elaborating on what type of automation (or
+lack thereof) can help get a sense of how robust a validator setup is. Many everyday actions or
+routine maintenance needs to be done, and automating this type of thing often helps mitigate human
+errors.
+
+#### Network Topology
+
+Does the Validator node have protection against Denial of Service attacks, and if so, how is that
+done? Outlining a desired network topology for a Validators infrastructure design will help
+Nominators understand how resilient their operations are to attacks. Some things to highlight are
+the usage of firewalls, VPNs, network segmentation, proxies, or other layers separation.
+
+#### Upgrading
+
+Both Polkadot and Kusama releases are published.
+[here](https://github.com/paritytech/polkadot-sdk/releases). Validators are expected to upgrade
+their nodes as soon as a new release comes. Although not every release is mandatory to upgrade, each
+new release usually has bug fixes, optimizations, new features, or other beneficial changes. It's in
+the best interest of the entire network that validators update their nodes in a timely fashion. This
+signals to nominators that a validator is timely care about their operations and is quick to adapt
+to necessary circumstances.
+
+It can also be helpful for Nominators to know how the Validator runs software and where they get new
+binaries. How do they get alerted for new releases? Do they receive updates from the matrix
+chatrooms? Do they have alerts for particular GitHub activities? Do they use the Debian/RPM
+packages? Do they use the Parity-provided GitHub binaries? Do they use Parity Docker images? Do they
+make their own Docker images? Do they build the binaries themselves? Validators often have their own
+build server for making binaries. If they take the extra steps to make these and do not rely on
```

Comment:
We do prefer the use of contractions for 'not' combinations. Readers are more likely to see the 'not' indicator if it is written "don't" or "won't" rather than as two words they might miss one if skimming, etc.